cmd/ctr/commands/tasks/metrics.go

```diff
@@ -73,7 +73,7 @@ var metricsCommand = cli.Command{
 		}
 		metric, err := task.Metrics(ctx)
 		if err != nil {
-			return nil
+			return err
```

Comment:
Looks like this was added in https://github.com/containerd/containerd/pull/2504 

@samuelkarp I don't see a mention in that PR of this being done on purpose, so I assume it's ok to change.

Reply:
I think I _probably_ did this because sometimes I saw a task with no metrics?  But all of `ctr` is effectively sample/debugging code, so I see no issue with making this change.